bin/bundler-search

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Search your bundle for the provided pattern
+#   Requires bundler 1.8+ for execution as a bundler subcommand.
+#   Examples:
+#     bundle search Kernal.warn
+#     bundle search current_user clearance
```

Comment:
If I wanted to search for a query that has multiple words, do I just quote it?

`bundle search "Change your password" clearance`

Might be worth adding to the example.